src/entities/tag.cpp

```diff
@@ -469,7 +482,8 @@ bool Tag::store() {
     if (this->id > 0) {
         query.prepare(
                 "UPDATE tag SET name = :name, priority = :priority, "
-                        "parent_id = :parentId, " + colorField + " = :color "
+                        "parent_id = :parentId, " + colorField + " = :color, "
+                        "created = datetime('now') "
```

Comment:
I think it's very misleading your your are using the column `created` as a update date, we should use a column `updated`